src/clients/repositories.rb

```diff
@@ -171,6 +171,7 @@ def PriorityToString(priority)
 
     def ReposFromService(service, input)
       input = deep_copy(input)
+      service = "" if service == :without
```

Comment:
`:without` is used at several places, we should define a constant for it.

(And `:without` symbol sounds a bit strange for me, I'd rather prefer `:no_service` or something like that.)